src/backend/game_config.ts

```diff
@@ -270,7 +271,8 @@ class GameConfigV0 extends GameConfig {
       if (isMac) {
         defaultSettings.wineCrossoverBottle = wineCrossoverBottle
       } else if (isLinux) {
-        defaultSettings.winePrefix = winePrefix || `${userHome}/.wine`
+        defaultSettings.winePrefix =
+          winePrefix || join(heroicDefaultWinePrefix, 'default')
```

Comment:
maybe we can add `join(heroicDefaultWinePrefix, 'default')` as a constant in constants.ts?